xcp/xmlunwrap.py

```diff
@@ -22,16 +22,19 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 """xmlunwrap - general methods to unwrap XML elements & attributes"""
+from xml.dom.minidom import Element
 
 import six
 
 class XmlUnwrapError(Exception):
     pass
 
-def getText(nodelist):
+def getText(element):
+    # type:(Element) -> str
```

Comment:
I think these would be better as type annotations which could eventually be checked by something like mypy.